custom-routes-vis/src/main/webapp/index.html

```diff
@@ -25,12 +25,15 @@ <h5>Origin Coordinates:</h5>
         <div id='origin-coordinates'></div><br>
         <button type="button" class="btn btn-success" onclick="showMarker('origin')" id="show-origin-marker">Place Marker</button>
         <button type="button" class="btn btn-danger hidden" onclick="hideMarker('origin')" id="hide-origin-marker">Delete Marker</button><br>
+        <div id='ios-url'></div><br>
       </div>
       <div class="col-sm-4">
         <h5>Destination Coordinates:</h5>
         <div id='destination-coordinates'></div><br>
         <button type="button" class="btn btn-success" onclick="showMarker('destination')" id="show-destination-marker">Place Marker</button>
         <button type="button" class="btn btn-danger hidden" onclick="hideMarker('destination')" id="hide-destination-marker">Delete Marker</button><br>
+        <div id='v1-android-url'></div><br>
+        <div id='v2-android-url'></div><br>
```

Comment:
Are the line breaks necessary?

Reply:
In this case, I thought it would be easier for users to press in the mobile mode to have space in between, what do y'all think?